pkgs/development/python-modules/suds/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPyPy,
+  pytestCheckHook,
+  python311Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "suds";
+  version = "1.1.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-HVz6dBFxk7JEpCM/JGxIPZ9BGYtEjF8UqLrRHE9knys=";
+  };
+
+  propagatedBuildInputs = with python311Packages; [
+    six
+  ];
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "suds" ];
+
+  meta = {
+    changelog = "https://github.com/suds-community/suds/blob/v${version}/CHANGELOG.md";
+    description = " Suds is a lightweight SOAP python client for consuming Web Services. A community fork of the jurko fork.";
```

```suggestion
    description = "Lightweight SOAP python client for consuming Web Services";
```

https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description